src/components/Main.js

```diff
@@ -110,31 +114,10 @@ export default function Main({ onHandleSubmit }) {
           <input id="flightTime" type="time" aria-label="time" />
         </div>
       </Time>
-      <Aircraft>
+      <AircraftStyle>
         <h2>Aircraft Model</h2>
-        <select
-          aria-label="select a aircraft model"
-          id="Aircraft"
-          onChange={(e) => setAircraft(e.target.value)}
-        >
-          <option value="--" aria-label="initial state"></option>
-          <option value="C172" aria-label="cessna 172R">
-            CESSNA 172R
-          </option>
-          <option value="B738" aria-label="boing 737">
-            B737-800
-          </option>
-          <option value="B74F" aria-label="boing 747">
-            B747-400F
-          </option>
-          <option value="B77L" aria-label="boing 777">
-            B777-200LR
-          </option>
-          <option value="SF50" aria-label="vision jet">
-            VISION JET
-          </option>
-        </select>
-      </Aircraft>
+        <Aircraft selectedAircraft={selectAircraft} />
```

Comment:
selectedAircraft does not sound like a function, but it is! Please rename it to something like onSelectAircraft

Reply:
Thanks for the feedback Felix, all done!